tests/integration_tests/saas/test_sentry_task.py

```diff
@@ -205,22 +207,53 @@ def test_sentry_access_request_task(
     )
 
 
+def _get_issues(
+    project: Dict[str, Any],
+    secrets: Dict[str, Any],
+    headers: Dict[str, Any],
+) -> Optional[Dict[str, Any]]:
+    response = requests.get(
+        f"https://{secrets['host']}/api/0/projects/{project['organization']['slug']}/{project['slug']}/issues/",
+        headers=headers,
+    )
+    if not response.ok or response.json() == []:
+        return None
+
+    return response.json()
+
+
 def sentry_erasure_test_prep(sentry_connection_config, db):
     sentry_secrets = sentry_connection_config.secrets
     # Set the assignedTo field on a sentry issue to a given employee
     token = sentry_secrets.get("erasure_access_token")
     issue_url = sentry_secrets.get("issue_url")
     sentry_user_id = sentry_secrets.get("user_id_erasure")
+    host = sentry_secrets.get("host")
 
     if not token or not issue_url or not sentry_user_id:
         # Exit early if these haven't been set locally
         return None, None, None
 
     headers = {"Authorization": f"Bearer {token}"}
     data = {"assignedTo": f"user:{sentry_user_id}"}
-    resp = requests.put(issue_url, json=data, headers=headers)
-    assert resp.status_code == 200
-    assert resp.json()["assignedTo"]["id"] == sentry_user_id
+    response = requests.put(issue_url, json=data, headers=headers)
+    assert response.ok
+    assert response.json()["assignedTo"]["id"] == sentry_user_id
```

Comment:
This assertion will error if `response.json()` doesn't have the expected type/keys. A slightly safer solution would be:
```suggestion
    assert isinstance(response.json(), dict)
    assert re["assignedTo"]["id"] == sentry_user_id
```

Reply:
This will still throw `NoneType` errors, it'd need to set a default on the first dict like:
```
assert response.json().get("assignedTo", {}).get("id") == sentry_user_id
```

Reply:
@galvana Looks like you missed a suggestion here?

Reply:
Updated